### PR TITLE
feat: Customizer Refresh Control PHP to ReactJS.

### PR DESCRIPTION
**Pull request by RajkiranBagal**
### Description
 - Convert Customizer Refresh from PHP to ReactJS.
 - Trello Task - https://trello.com/c/qlEbuDEW/21-ast-customizer-refresh

### Types of changes
New feature (non-breaking change which adds functionality) 

### How has this been tested?
 - Check Backward Compatibility.
 - Check if all options are working and saving to database.

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] I've included any necessary tests 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by premanshup** [comment]
@RajkiranBagal This PR should be opened for the Astra Pro plugin. Please verify once.

File: inc/customizer/react/src/customizer-refresh/control.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import CustomizerRefreshComponent from './customizer-refresh-component.js';
```
Comment (premanshup) [inline inc/customizer/react/src/customizer-refresh/control.js:1]: @RajkiranBagal We should move this to the Astra Pro plugin. Or maybe we can keep it as it is as the HTML is not much and that too we are using this only once. Let me know your thoughts.

Comment (RajkiranBagal) [reply inc/customizer/react/src/customizer-refresh/control.js:1]: @premanshup Sir
As to give this PR to Astra Pro will have to do all the setup for the React in Astra Pro.

So, I added that in the Astra theme. There is no issue for functionality.

Also, Yes there is not much HTML only one anchor tag, So we can keep it as it is.

But to maintain consistency I converted that to react.

Comment (premanshup) [reply inc/customizer/react/src/customizer-refresh/control.js:1]: @RajkiranBagal As discussed, keeping this control in Addon only.

**Comment by premanshup** [comment]
Closing this PR as this is added from Addon.

**Pull request closed without merging**